backend/src/main/java/net/gunn/elimination/routes/SSEController.java

```diff
@@ -94,12 +97,23 @@ public SseEmitter scoreboardStream(@RequestParam(defaultValue = "20") int limit)
 	}
 
 	@GetMapping(value = "/game/eliminations", produces = "text/event-stream")
-	public SseEmitter kills() {
+	public SseEmitter kills() throws IOException {
+		var em = emf.createEntityManager();
 		var emitter = new SseEmitter(-1L);
 
 		killEmitters.add(emitter);
```

Comment:
this should probably happen after we do .onCompetion

Reply:
this isn't the case for the other SSE endpoints, should i move all of them?